platform/solr/platform-solr-server-standalone/src/main/webapp/old.html

```diff
@@ -79,8 +79,9 @@
                 
       </div>
 
-      <div class="other-ui-link">
-        Try <a class="ul" href="/solr/index.html">New UI</a><a target="_blank" href="http://wiki.apache.org/solr/AngularUI">&nbsp;<span class="help"></span></a>
+      <div class="old-ui-warning">
+        THIS USER INTERFACE IS DEPRECATED. Please use the current UI <a class="ul" href="/solr/">here</a>
```

Comment:
Guess this answers my question. :) What's the value in keeping the old ui though, vs simply redirecting it to the new ui (maybe still with a warning)?